modules/aleph/lib/AlephClient/AlephClient.class.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * @file
+ * Provides a client for the Axiell Aleph library information webservice.
+ */
+
+require __DIR__ . '../../../vendor/autoload.php';
```

Comment:
```grep -r 'guzzle' .``` in the root shows that guzzle is used in two modules and then theres a library (I removed some of the output for convenience):

```
./web/profiles/ding2/modules/fbs/composer.json:        "guzzlehttp/guzzle": "5.2.0"
./web/profiles/ding2/modules/aleph/composer.json:        "guzzlehttp/guzzle": "^6.2"
./web/profiles/ding2/modules/bpi/lib/bpi-client/composer.json:        "guzzlehttp/guzzle": "^6.2"
./web/profiles/ding2/ding2.make:libraries[guzzle][download][tag] = "6.2.2"
./web/profiles/ding2/ding2.make:libraries[guzzle][destination] = "libraries"
./web/profiles/ding2/ding2.make:libraries[promises][download][url] = "https://github.com/guzzle/promises.git"
./web/profiles/ding2/ding2.make:libraries[psr7][download][url] = "https://github.com/guzzle/psr7.git"
```